python/grapher.go

```diff
@@ -37,14 +37,15 @@ func NewGraphContext(unit *unit.SourceUnit) *GraphContext {
 // been installed (via pip or `python setup.py install`).
 func (c *GraphContext) Graph() (*graph.Output, error) {
 	programMode := os.Getenv("IN_DOCKER_CONTAINER") == ""
-	tc, err := toolchain.Lookup("sourcegraph.com/sourcegraph/srclib-python")
-	if err != nil {
-		return nil, err
-	}
 	pipBin := "pip"
 	pythonBin := "python"
 
 	if programMode {
+		tc, err := toolchain.Lookup("sourcegraph.com/sourcegraph/srclib-python")
```

Comment:
Is this snippet of code necessary? We are inside the Python toolchain, which should be agnostic of whether it's on the srclib path or not. Maybe we can just delete this.